modular_skyrat/modules/modular_items/lewd_items/code/lewd_chemistry/reagents/_aphrodisiac.dm

```diff
@@ -39,9 +39,9 @@
 	/// Smallest girth the chem can make a mob's penis
 	var/penis_minimum_girth = 2
 	/// How much to reduce the size of the penis each time it's run
-	var/penis_size_reduction_step = 2
```

Comment:
This being set to 2 was causing inconsistent behavior because an even numbered size is possible. 

E.g. case: size 12 penis being reduces by 2 each time. 12->10->8->6->3->2->0
We now have a size 0 . 
If it had started out at size 13, it'd only be able to be reduced to 1. Hence the change, to allow for consistent shrinkage.